grid-cli/docs/kubernetes.md

```diff
@@ -37,9 +39,18 @@ Example:
 
 ```console
 $ tfcmd deploy kubernetes -n kube --ssh ~/.ssh/id_rsa.pub --master-node 14 --workers-number 2 --workers-node 14
-4:21PM INF deploying network
-4:22PM INF deploying cluster
-4:22PM INF master planetary ip: 300:e9c4:9048:57cf:504f:c86c:9014:d02d
+11:43AM INF starting peer session=tf-1510734 twin=192
+11:43AM INF deploying network
+11:43AM INF deploying cluster
+11:43AM INF master wireguard ip: 10.20.2.2
+11:43AM INF master planetary ip: 300:e9c4:9048:57cf:d73d:eb4c:7a6d:503
+11:43AM INF master mycelium ip: 423:16f5:ca74:b600:ff0f:9ee:d57e:827a
+11:43AM INF worker1 wireguard ip: 10.20.2.3
+11:43AM INF worker0 wireguard ip: 10.20.2.3
```

Comment:
is it normal to have same ips for workers?

Reply:
nope not normal this is a bug

Reply:
created an issue for that #909